txs_toml.py

```diff
@@ -0,0 +1,77 @@
+import toml
+import os
+import subprocess
+import sys
+import decimal
+
+def compute_naan_total_supply(balances_toml):
+    return sum(map(decimal.Decimal, balances_toml["token"]["NAAN"].values()))
+
+def source_has_nam_balance(balances_toml, source):
+    return balances_toml["token"]["NAAN"].get(source)
+
+def main():
+    if len(sys.argv) < 2:
+        print(f'Usage: python {sys.argv[0]} <genesis_files_path> <txs_path>')
+        return
+
+    genesis_files_path = sys.argv[1]
+    txs_path = sys.argv[2]
+
+    balances_toml = toml.load(os.sep.join([genesis_files_path, 'balances.toml']))
+    big_toml = {"established_account": [], "validator_account": [], "bond": []}
+
+    decimal.getcontext().prec = 256
+
+    initial_supply = compute_naan_total_supply(balances_toml)
+
+    for file in filter(lambda path: path[-5:] == '.toml', os.listdir(txs_path)):
+        mini_toml = toml.load(os.sep.join([txs_path, file]))
+
+        big_toml["established_account"].extend(mini_toml["established_account"])
+        big_toml["validator_account"].extend(mini_toml["validator_account"])
+        big_toml["bond"].extend(mini_toml["bond"])
+
+        amount = decimal.Decimal(mini_toml["bond"][0]["amount"])
+        source = mini_toml["bond"][0]["source"]
+
```

Comment:
This should ensure that every account has at least 10 NAAN - ie even if you bonded it all, then the script gives you a bit extra